docs/USER_GUIDE.md

```diff
@@ -330,6 +355,20 @@ requests using the AWS CLI:
    curl $API_URL/v1/queue -H "Authorization: Bearer $ID_TOKEN"
    ```
 
+#### IAM
+
+IAM authentication for API requests uses the
+[Signature Version 4 signing process](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).
+Add the resulting signature to the **Authorization** header when making requests
+to the API.
+
+Use the Sigv4 process linked above to generate the Authorization header value
+and then call the API as normal:
+
+```sh
+curl $API_URL/v1/queue -H "Authorization: $Sigv4Auth"
+```
+
```

Comment:
The line below this (372) should be under the Cognito section?

Reply:
Yep, thanks.